docs/source/development-guide/getting-started.rst

```diff
@@ -0,0 +1,29 @@
+.. _getting-started:
+
+Getting Started
+===============
+
+Installing requirements
+-----------------------
+
+``imap-processing`` uses :ref:`poetry-link` for dependency management. Check out our :ref:`style guide <poetry-environment>` for specific IMAP Poetry usage.
+
+To install using Poetry for development:
+
+.. code-block:: bash
+
+    poetry self add poetry-dynamic-versioning
+    poetry install --all-extras
+
+
+Using IMAP processing
+---------------------
+
+The main way to run IMAP processing locally is through the :ref:`cli`.
+
+Our tests are run using pytest:
+
+.. code-block:: bash
+
+    poetry shell
+    pytest
```

Comment:
can you move this into [this page](https://imap-processing.readthedocs.io/en/latest/development-guide/style-guide/poetry-environment.html). Then use this page to point to other pages that a new user may need to get started. Like the help page.

Reply:
Ok, what help page do you mean?

Reply:
I kind of think this should stay here because it is not a poetry specific move, but it is a minor thing that new people might want to use.

Reply:
This page: https://lasp.colorado.edu/galaxy/pages/viewpage.action?pageId=193401231

Reply:
Ah yes, good idea!